Codeinwp/neve
# Meta custom fields

### Summary
This PR replaces the old meta control with the repeater control. It also adds the "Hide meta on mobile" feature.

### Will affect the visual aspect of the product
NO

### Screenshots 
<img width="301" alt="Screenshot 2022-05-25 at 13 18 19" src="https://user-images.githubusercontent.com/9929553/170240077-d51903a7-5d7c-440b-abfc-0453419aad19.png">

### Test instructions
- Start by testing just with Neve active.
- The meta fields should work the same as before.
- If you previously set a meta order and elements visibility, with this PR the order should maintain
- Check if the hide on mobile feature works
- Test if other settings from meta, like separator, still work
- Please test with various combinations like Old Neve / New Neve Pro and New Neve / Old Neve Pro. Here is the PR for New Neve PRO https://github.com/Codeinwp/neve-pro-addon/pull/2002.


Closes Codeinwp/neve-pro-addon#1651, #3009
